src/logger.ts

```diff
@@ -31,7 +31,7 @@ export interface LogEntry {
 
 /**
  * Writes a `LogEntry` to `stdout`/`stderr` (depending on severity).
- * @param entry The LogEntry including severity, message, and any additional structured metadata.
+ * @param entry The `LogEntry` including severity, message, and any additional structured metadata.
```

Comment:
Will the `LogEntry` type show up in the reference? Can we link to it? Or maybe that happens automatically since it's an argument for the function?

Reply:
Yup!  TypeDoc creates a link for that.